### ISSUE TITLE
OpenDKIM not working on CentOS 7 because of bad option added to /etc/opendkim.conf

### ISSUE THREAD

**Issue by almico** [modoboa/modoboa-installer#308]
# Impacted versions
Modoboa 1.14.0

* Distribution: Centos
* Codename: Centos 7
* Arch: 64 Bits
* Database: PostgreSQL

# Steps to reproduce

Purchase a new server then run the installer.
I found the following error message in `/var/log/maillog`:

```
Mar 29 11:51:26 test postfix/smtpd[17639]: warning: connect to Milter service inet:127.0.0.1:12345: Connection refused
```

It was due to OpenDKIM being unable to start.
This was the log from opendkim service:

```
Mar 29 10:25:10 test systemd[1]: Starting DomainKeys Identified Mail (DKIM) Milter...
Mar 29 10:25:10 test opendkim[28566]: opendkim: /etc/opendkim.conf: configuration error at line 94: unrecognized parameter
Mar 29 10:25:10 test systemd[1]: opendkim.service: control process exited, code=exited status=78
Mar 29 10:25:10 test systemd[1]: Failed to start DomainKeys Identified Mail (DKIM) Milter.
Mar 29 10:25:10 test systemd[1]: Unit opendkim.service entered failed state.
Mar 29 10:25:10 test systemd[1]: opendkim.service failed.
```

I noticed that `/etc/opendkim.conf` contained, at its end, the following line:

```
SOCKET="inet:12345@localhost"
```

Removing it, saving the file and restarting opendkim service did the trick.
Basically, that `SOCKET="inet:12345@localhost"` is both using the wrong syntax and redundant because that file already contains `Socket inet:12345@localhost`.

# Expected behavior

The installer should not add `SOCKET="inet:12345@localhost"` to `/etc/opendkim.conf`.
